PROTOCOL.md

```diff
@@ -995,6 +1065,178 @@ Within the checkpoint, the `remove` struct does not contain the `stats` and `tag
 
 Refer to the [appendix](#checkpoint-schema) for an example on the schema of the checkpoint.
 
+Delta supports two checkpoint specs and three kind of checkpoint naming schemes.
+
+### Checkpoint Specs
+Delta supports following two checkpoint specs:
+
+#### V2 Spec
+This checkpoint spec allows putting [add and remove file](#Add-File-and-Remove-File) in the
+[sidecar files](#sidecar-files). This spec can be used only when [v2 checkpoint table feature](#v2-checkpoint-table-feature) is enabled.
+Checkpoints following V2 spec have the following structure:
+- Each v2 spec checkpoint includes exactly one [Checkpoint Metadata](#checkpoint-metadata) action.
+- Remaining rows in the V2 spec checkpoint refers the other actions mentioned [here](#checkpoints-1)
```

Comment:
```suggestion
- Remaining rows in the V2 spec checkpoint refer to the other actions mentioned [here](#checkpoints-1)
```

Reply:
Also, "here" isn't a super helpful link title... but I don't know a better name given that we have two sections about checkpointing right now?